app/assets/sass/pages/card-details.scss

```diff
@@ -59,3 +59,11 @@
   cursor: pointer;
   font-size: 19px;
 }
+
+#footer .footer-meta .footer-meta-inner {
+  .merchant-details,
+  .merchant-details-line-1,
```

Comment:
have you tried to assign the `display: block` only to merchant-details? It should be inherited by the `<li>`s, so you might be able to simplify the code

Reply:
@simad it's needs to be this specific to override the really specific code coming from govuk_template 🙄

Reply:
Thanks @simad .
I've tried that initially, but unfortunately it doesn't work, because we have more specific rule `#footer .footer-meta .footer-meta-inner ul li` in [_footer.scss#L68](https://github.com/alphagov/govuk_template/blob/e9ae7ff045e0adf552aeed509be5ef174be325aa/source/assets/stylesheets/_footer.scss#L68) which overrides less specific one: `#footer .footer-meta .footer-meta-inner .merchant-details`

Reply:
gotcha, thanks